security/middleware.py

```diff
@@ -692,8 +694,9 @@ def process_response(self, request, response):
         enforcement or report-only headers in all currently used variants.
         """
         # choose headers based enforcement mode
+        is_ie = user_agent_parser.ParseUserAgent(request.META['HTTP_USER_AGENT'])['family'] == 'IE'
```

Comment:
No, because the UA string for Edge is "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.71 Safari/537.36 Edge/12.0" ([link](http://www.davevoyles.com/microsofts-spartan-browser-uses-chrome-ua-string/)) which parses to family='Edge':
```
>>> from ua_parser import user_agent_parser
>>> parsed_ua_str = user_agent_parser.ParseUserAgent('Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.71 Safari/537.36 Edge/12.0')
>>> parsed_ua_str
{'major': '12', 'minor': '0', 'family': 'Edge', 'patch': None}
```